data_structures/route_parameters.cpp

```diff
@@ -36,7 +36,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 RouteParameters::RouteParameters()
     : zoom_level(18), print_instructions(false), alternate_route(true), geometry(true),
-      compression(true), deprecatedAPI(false), uturn_default(false), classify(false),
+      compression(true), deprecatedAPI(false), uturn_default(false), classify(false), mapped_points(true),
```

Comment:
For really huge matrices you should avoid going over the network (uncompressed) anyway. You can take a look at our `node-osrm` bindings. Since this is all 100% numerical values you can get a way with about 4 bytes per entry (just save the raw matrix and the size).